frameworks/PHP/phalcon/public/index-micro.php

```diff
@@ -1,114 +1,109 @@
 <?php
 
-try {
+use Phalcon\Db\Adapter\Pdo\Mysql;
+use Phalcon\Db\Enum;
+use Phalcon\Mvc\Micro;
+use Phalcon\Mvc\View;
+use Phalcon\Mvc\View\Engine\Volt;
+use Phalcon\Exception as PhalconException;
+
+use function header;
+use function json_encode;
```

Comment:
Please delete these two lines, they are useless.
It is trying to add non-namespaced imports to non-namespaced class, so by default it automatically inherits the global symbol table.

Check the error:
https://github.com/TechEmpower/FrameworkBenchmarks/pull/6443/checks?check_run_id=2104369951#step:9:646

Reply:
Done.